src/NUnitConsole/nunit3-console/ResultReporter.cs

```diff
@@ -118,16 +118,21 @@ public void WriteSummaryReport()
 
             WriteSummaryCount("   Tests run: ", Summary.RunCount);
             WriteSummaryCount(", Passed: ", Summary.PassCount);
-            WriteSummaryCount(", Errors: ", Summary.ErrorCount, ColorStyle.Error);
-            WriteSummaryCount(", Failures: ", Summary.FailureCount, ColorStyle.Failure);
+            WriteSummaryCount(", Failed: ", Summary.FailedCount, ColorStyle.Failure);
             WriteSummaryCount(", Inconclusive: ", Summary.InconclusiveCount);
+            WriteSummaryCount(", Skipped: ", Summary.SkipCount);
```

Comment:
@rprouse You are absolutely right. I have implemented the new property in my last commit. The tests are running at the moment, so please hang on a couple of minutes in order to see the latest status.